tests/e2e-tests.sh

```diff
@@ -0,0 +1,19 @@
+#! /bin/bash
+
+# e2e tests basic for operator
+
+# if we run in travis, we assume that kind is used.
+# if we run locally e2e tests default is kubeconfig (not binded to kind)
+if [ "$TRAVIS" = true ] ; then
```

Comment:
I would prefer this to be set in travis script, which knows it is using kind and therefore, can set the KUBECONFIG variable in the proper way. Also, if we later decide to change how we setup the cluster in travis we don't need to change anything here. Let's keep concerns apart.

Reply:
I agree with @pablochacin : I think we should not try to be too smart here and add this kind of dependency. The `KUBECONFIG` should be already defined (maybe we should fail otherwise) and we should just run the tests.

Reply:
thx adressed in new commit